main/webapp/modules/core/langs/translation-en.json

```diff
@@ -518,7 +518,7 @@
     "core-views/word-facet": "Word facet",
     "core-views/add-id-column": "Add entity identifiers column",
     "core-views/add-id-column2": "Add a column containing entity identifiers from items",
-    "core-views/add-id-col": "Add column containing entity identifiers on column",
+    "core-views/add-id-col": "Add column containing entity identifiers on $1 {{plural:$1|column}}",
```

Comment:
Why use `{{plural}}` here? Isn't plural intended to be used for numbers? `$1` represents a string, not a number here.

Reply:
I see. I thought that was a i18n parameter, my bad. Should I remove it all and keep just $1?

Reply:
`Add column containing entity identifiers on column $1` should do: `$1` will just be replaced by the column name.

Reply:
Yep that's what I meant :+1:

Reply:
Fixed :)